src/cluster_slot_stats.c

```diff
@@ -148,10 +148,11 @@ static void clusterSlotStatsUpdateNetworkBytesOutForReplication(long long len) {
     client *c = server.current_client;
     if (c == NULL || !canAddNetworkBytesOut(c)) return;
 
+    uint64_t rlen = len*listLength(server.replicas);
```

Comment:
Would prefer explicit conversion than implicit conversion.

```suggestion
    uint64_t rlen = (uint64_t)llabs(len) * listLength(server.replicas);
```

Reply:
We can't do the llabs here since that would turn negative numbers into positive numbers and we need to handle negative and positive numbers separately. Positive numbers increase the counter while negative numbers decrease the counter (for the special handling of the `SELECT` command) where wo do the explicit assert check.